static/app/views/issueDetails/actions/index.spec.tsx

```diff
@@ -253,58 +253,53 @@ describe('GroupActions', function () {
     });
 
     it('delete for issue platform', async () => {
+      const router = RouterFixture();
```

Comment:
I'm updating this test to match what the normal issues do.
I will point out the difference with the other test.